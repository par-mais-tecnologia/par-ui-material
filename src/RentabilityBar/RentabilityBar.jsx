import React from 'react'
import PropTypes from 'prop-types'
import { withStyles } from '@material-ui/core'

const MainBoxClasses = 'flex items-center justify-center shadow-3'
const SubBoxClasses = 'flex items-center justify-between'
const ColumnsClasses = 'flex flex-column ph2'
const styles = theme => (theme.rentabilityBar)

const RentabilityBar = (props) => {
  const { classes } = props
  return (
    <div className={[ MainBoxClasses, classes.mainBoxStyles ].join(' ')}
    >
      <div className={[SubBoxClasses, classes.subBoxStyles].join(' ')}>
        <div className={ColumnsClasses}>
          <h1 className={classes.title}>
            {props.firstTitle}
          </h1>
          <p className={
            props.firstValue && props.firstValue > 0
              ? classes.positiveState : classes.neutralState
          }>
            <small>
              {
                props.firstValue && props.firstValue > 0
                  ? '+' : ''
              }
            </small>
            {props.firstValue}
          </p>
        </div>
        <div className={ColumnsClasses}>
          <h1 className={classes.title}>
            {props.secondTitle}
          </h1>
          <p className={
            props.secondValue && props.secondValue > 0
              ? classes.positiveState : classes.neutralState
          }>
            <small>
              {
                props.secondValue && props.secondValue > 0
                  ? '+' : ''
              }
            </small>
            {props.secondValue}
          </p>
        </div>
        <div className={ColumnsClasses}>
          <h1 className={classes.title}>
            {props.thirthTitle}
          </h1>

          <p className={
            props.thirthValue && props.thirthValue > 0
              ? classes.positiveState : classes.neutralState
          }>
            <small>
              {
                props.thirthValue && props.thirthValue > 0
                  ? '+' : ''
              }
            </small>
            {props.thirthValue}
          </p>
        </div>
      </div>
    </div>
  )
}

RentabilityBar.propTypes = {
  firstValue: PropTypes.number,
  secondValue: PropTypes.number,
  thirthValue: PropTypes.number
}

export default withStyles(styles)(RentabilityBar)
