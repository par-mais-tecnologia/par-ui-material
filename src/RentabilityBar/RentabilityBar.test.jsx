import React from 'react'
import { configure, shallow } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import { SeeTheme, MuiThemeProvider } from '../'

import RentabilityBar from './'

configure({ adapter: new Adapter() })

export const test = describe('RentabilityBar', () => {
  const getShallow = () => (
    <MuiThemeProvider theme={SeeTheme}>
      <RentabilityBar
        header={[
          {
            title: 'Rentabilidade em R$',
            value: 100.00,
            type: 'currency'
          },
          {
            title: 'Rentabilidade em %',
            value: -0.01
          },
          {
            title: 'cdi',
            value: 0.4786554
          }
        ]}
      />
    </MuiThemeProvider>
  )

  it('shallow RentabilityBar', () => {
    const wrapper = shallow(
      <MuiThemeProvider theme={SeeTheme}>
        <RentabilityBar />
      </MuiThemeProvider>
    )
    expect(wrapper).toBeTruthy()
  })

  it('shallow RentabilityBar match Snapshot', () => {
    const wrapper = shallow(getShallow())
    expect(wrapper).toMatchSnapshot()
  })
})
