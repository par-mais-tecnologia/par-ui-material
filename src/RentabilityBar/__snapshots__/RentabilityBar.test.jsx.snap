// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RentabilityBar shallow RentabilityBar match Snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <MuiThemeProviderOld
    theme={
      Object {
        "breakpoints": Object {
          "between": [Function],
          "down": [Function],
          "keys": Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ],
          "only": [Function],
          "up": [Function],
          "values": Object {
            "lg": 1280,
            "md": 960,
            "sm": 600,
            "xl": 1920,
            "xs": 0,
          },
          "width": [Function],
        },
        "buttonStyle": Object {
          "contained": Object {
            "&$contained:focus": Object {
              "backgroundColor": "white",
              "color": "rgba(99, 43, 125, 1)",
            },
            "&$contained:hover": Object {
              "backgroundColor": "#491760",
            },
            "backgroundColor": "rgba(99, 43, 125, 1)",
            "color": "white",
          },
          "outlined": Object {
            "&$outlined:focus": Object {
              "backgroundColor": "#F0F0F0",
            },
            "&$outlined:hover": Object {
              "backgroundColor": "#F0F0F0",
            },
            "backgroundColor": "white",
            "borderStyle": "hidden solid solid solid",
            "boxShadow": "0px 1px #C1C1C1",
            "color": "rgba(99, 43, 125, 1)",
          },
        },
        "direction": "ltr",
        "financialPanel": Object {
          "box": Object {
            "height": "1rem",
            "marginRight": "1rem",
            "width": "1rem",
          },
          "close": Object {
            "height": 0,
            "overflow": "hidden",
          },
          "detailsContent": Object {
            "color": "#636363",
            "fontSize": "1rem",
          },
          "detailsLine": Object {
            "marginBottom": "1rem",
          },
          "detailsTitle": Object {
            "color": "#9C9C9C",
            "fontSize": ".8rem",
            "textTransform": "uppercase",
          },
          "flexBoxBetween": Object {
            "alignItems": "center",
            "display": "flex",
            "justifyContent": "space-between",
            "width": "100%",
          },
          "flexBoxStart": Object {
            "display": "flex",
            "justifyContent": "flex-start",
            "minWidth": "25%",
          },
          "header": Object {
            "borderBottom": "1px solid #d4d4d4",
            "boxShadow": "none",
          },
          "name": Object {
            "color": "#636363",
            "fontSize": "1rem",
          },
          "open": Object {
            "height": "auto",
            "margin": "1rem 0",
          },
          "root": Object {
            "width": "100%",
          },
          "value": Object {
            "color": "#636363",
            "fontSize": "1rem",
            "marginRight": "1rem",
          },
          "walletComposition": Object {
            "color": "#636363",
            "fontSize": "1rem",
          },
        },
        "lineGraph": Object {
          "axis": Object {
            "fontFamily": "Roboto Regular, sans-serif",
            "fontSize": "14px",
          },
          "buttonActive": Object {
            "&:hover": Object {
              "background": "#5EB8C0",
            },
            "background": "#5EB8C0",
            "border": "none",
            "borderRadius": "20px",
            "boxShadow": "none",
            "color": "#fff",
            "transition": "all .5s ease",
          },
          "buttonDefault": Object {
            "&:hover": Object {
              "background": "transparent",
            },
            "background": "transparent",
            "border": "1px solid #9C9C9C",
            "borderRadius": "20px",
            "boxShadow": "none",
            "color": "#9C9C9C",
            "transition": "all .5s ease",
          },
          "buttonsRow": Object {
            "justifyContent": "space-evenly",
            "margin": "2rem",
            "minWidth": "15rem",
          },
          "buttonsWrapper": Object {
            "margin": "2rem",
          },
          "filterTabsRow": Object {
            "borderBottom": "#E2E2E2 solid 1px",
            "margin": "2rem",
          },
          "tabLabel": Object {
            "color": "#632B7D",
          },
          "tabsIndicator": Object {
            "backgroundColor": "#632B7D",
          },
          "tooltip": Object {
            "fill": "#333333",
            "opacity": "0.75",
            "transform": "translate(5px,-70px)",
          },
          "tooltipCircle": Object {
            "fill": "#632B7D",
          },
          "tooltipElement": Object {
            "transition": "all .0s ease-in",
          },
          "tooltipLine": Object {
            "stroke": "#636363",
            "strokeWidth": "1px",
          },
          "tooltipWrapper": Object {
            "background": "rgba(99, 43, 125, .8)",
            "borderRadius": "3px",
            "color": "#fff",
            "fontWeight": "14px",
            "minWidth": "10rem",
            "padding": "15px",
          },
        },
        "menu": Object {
          "menuItem": Object {
            "&:focus": Object {
              "backgroundColor": "#F0F0F0",
              "color": "#347A7C",
            },
            "&:hover": Object {
              "backgroundColor": "white",
              "color": "#347A7C",
            },
            "color": "#9C9C9C",
          },
        },
        "mixins": Object {
          "gutters": [Function],
          "toolbar": Object {
            "@media (min-width:0px) and (orientation: landscape)": Object {
              "minHeight": 48,
            },
            "@media (min-width:600px)": Object {
              "minHeight": 64,
            },
            "minHeight": 56,
          },
        },
        "overrides": Object {
          "MuiButton": Object {
            "outlinedSecondary": Object {
              "&:hover": Object {
                "border": "1px solid transparent",
              },
              "border": "1px solid transparent",
            },
          },
          "MuiExpansionPanelDetails": Object {
            "root": Object {
              "flexDirection": "column",
            },
          },
          "MuiFormControl": Object {
            "marginNormal": Object {
              "marginBottom": "10px",
              "marginTop": 0,
            },
          },
          "MuiFormHelperText": Object {
            "required": Object {
              "color": "#BA2B31",
              "fontFamily": "Roboto Regular, sans-serif",
              "fontSize": "14px",
              "marginTop": 0,
            },
          },
          "MuiFormLabel": Object {
            "asterisk": Object {
              "display": "none",
            },
            "root": Object {
              "&$error": Object {
                "color": "#BA2B31",
              },
              "&$focused": Object {
                "color": "#019193",
              },
            },
          },
          "MuiInput": Object {
            "underline": Object {
              "&$error:after": Object {
                "borderBottomColor": "#BA2B31",
              },
              "&:after": Object {
                "borderBottomColor": "#019193",
              },
              "&:before": Object {
                "borderBottom": "1px solid #9C9C9C",
              },
              "&:hover:not($disabled):not($focused):not($error):before": Object {
                "borderBottomColor": "#5EB8C0",
              },
            },
          },
          "MuiInputBase": Object {
            "input": Object {
              "caretColor": "#019193",
              "color": "#636363",
              "fontFamily": "Roboto Light, sans-serif",
              "fontSize": "16px",
              "padding": "10px 0 7px",
            },
            "inputType": Object {
              "height": "2rem",
            },
          },
          "MuiInputLabel": Object {
            "root": Object {
              "color": "#9C9C9C",
              "fontFamily": "Roboto Regular, sans-serif",
              "fontSize": "14px",
            },
            "shrink": Object {
              "transform": "translate(0, 1.5px) scale(1)",
            },
          },
          "MuiPickersSlideTransition": Object {
            "transitionContainer": Object {
              "fontFamily": "Roboto Light",
              "fontSize": "12px",
            },
          },
        },
        "palette": Object {
          "action": Object {
            "active": "rgba(0, 0, 0, 0.54)",
            "disabled": "rgba(0, 0, 0, 0.26)",
            "disabledBackground": "rgba(0, 0, 0, 0.12)",
            "hover": "rgba(0, 0, 0, 0.08)",
            "hoverOpacity": 0.08,
            "selected": "rgba(0, 0, 0, 0.14)",
          },
          "augmentColor": [Function],
          "background": Object {
            "default": "#fafafa",
            "paper": "#fff",
          },
          "common": Object {
            "black": "#000",
            "white": "#fff",
          },
          "contrastThreshold": 3,
          "divider": "rgba(0, 0, 0, 0.12)",
          "error": Object {
            "contrastText": "#fff",
            "dark": "#d32f2f",
            "light": "#e57373",
            "main": "#f44336",
          },
          "getContrastText": [Function],
          "grey": Object {
            "100": "#f5f5f5",
            "200": "#eeeeee",
            "300": "#e0e0e0",
            "400": "#bdbdbd",
            "50": "#fafafa",
            "500": "#9e9e9e",
            "600": "#757575",
            "700": "#616161",
            "800": "#424242",
            "900": "#212121",
            "A100": "#d5d5d5",
            "A200": "#aaaaaa",
            "A400": "#303030",
            "A700": "#616161",
          },
          "primary": Object {
            "contrastText": "rgba(0, 0, 0, 0.87)",
            "dark": "rgb(65, 128, 134)",
            "light": "rgb(126, 198, 204)",
            "main": "#5EB8C0",
          },
          "secondary": Object {
            "contrastText": "#fff",
            "dark": "#c51162",
            "light": "#ff4081",
            "main": "#f50057",
          },
          "text": Object {
            "disabled": "rgba(0, 0, 0, 0.38)",
            "hint": "rgba(0, 0, 0, 0.38)",
            "primary": "rgba(0, 0, 0, 0.87)",
            "secondary": "rgba(0, 0, 0, 0.54)",
          },
          "tonalOffset": 0.2,
          "type": "light",
        },
        "parallax": Object {
          "content": Object {
            "height": "100vh",
            "position": "relative",
          },
          "mobile": Object {
            "height": "100vh",
          },
          "parallaxContainer": Object {
            "height": "100vh",
            "overflowX": "hidden",
            "overflowY": "auto",
            "perspective": "1px",
          },
          "parallaxContent": Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
            "transform": "translateZ(-1px) scale(2.5)",
          },
          "parallaxSection": Object {
            "height": "60vh",
            "position": "relative",
          },
        },
        "props": Object {},
        "rentabilityBar": Object {
          "mainBoxStyles": Object {
            "backgroundColor": "#F0F0F0",
            "borderRadious": "20px",
            "padding": "1rem 0",
          },
          "negativeState": Object {
            "color": "#BA2B31",
            "fontFamily": "Roboto Regular",
            "fontSize": "16px",
            "fontWeight": "300",
            "margin": "0",
            "textTransform": "uppercase",
          },
          "neutralState": Object {
            "color": "#636363",
            "fontFamily": "Roboto Regular",
            "fontSize": "16px",
            "fontWeight": "300",
            "margin": "0",
            "textTransform": "uppercase",
          },
          "positiveState": Object {
            "color": "#7FBD42",
            "fontFamily": "Roboto Regular",
            "fontSize": "16px",
            "fontWeight": "300",
            "margin": "0",
            "textTransform": "uppercase",
          },
          "subBoxStyles": Object {
            "minWidth": "75%",
          },
          "title": Object {
            "color": "#9C9C9C",
            "fontFamily": "Roboto Regular",
            "fontSize": "14px",
            "fontWeight": "300",
            "textTransform": "uppercase",
          },
        },
        "shadows": Array [
          "none",
          "0px 1px 3px 0px rgba(0, 0, 0, 0.2),0px 1px 1px 0px rgba(0, 0, 0, 0.14),0px 2px 1px -1px rgba(0, 0, 0, 0.12)",
          "0px 1px 5px 0px rgba(0, 0, 0, 0.2),0px 2px 2px 0px rgba(0, 0, 0, 0.14),0px 3px 1px -2px rgba(0, 0, 0, 0.12)",
          "0px 1px 8px 0px rgba(0, 0, 0, 0.2),0px 3px 4px 0px rgba(0, 0, 0, 0.14),0px 3px 3px -2px rgba(0, 0, 0, 0.12)",
          "0px 2px 4px -1px rgba(0, 0, 0, 0.2),0px 4px 5px 0px rgba(0, 0, 0, 0.14),0px 1px 10px 0px rgba(0, 0, 0, 0.12)",
          "0px 3px 5px -1px rgba(0, 0, 0, 0.2),0px 5px 8px 0px rgba(0, 0, 0, 0.14),0px 1px 14px 0px rgba(0, 0, 0, 0.12)",
          "0px 3px 5px -1px rgba(0, 0, 0, 0.2),0px 6px 10px 0px rgba(0, 0, 0, 0.14),0px 1px 18px 0px rgba(0, 0, 0, 0.12)",
          "0px 4px 5px -2px rgba(0, 0, 0, 0.2),0px 7px 10px 1px rgba(0, 0, 0, 0.14),0px 2px 16px 1px rgba(0, 0, 0, 0.12)",
          "0px 5px 5px -3px rgba(0, 0, 0, 0.2),0px 8px 10px 1px rgba(0, 0, 0, 0.14),0px 3px 14px 2px rgba(0, 0, 0, 0.12)",
          "0px 5px 6px -3px rgba(0, 0, 0, 0.2),0px 9px 12px 1px rgba(0, 0, 0, 0.14),0px 3px 16px 2px rgba(0, 0, 0, 0.12)",
          "0px 6px 6px -3px rgba(0, 0, 0, 0.2),0px 10px 14px 1px rgba(0, 0, 0, 0.14),0px 4px 18px 3px rgba(0, 0, 0, 0.12)",
          "0px 6px 7px -4px rgba(0, 0, 0, 0.2),0px 11px 15px 1px rgba(0, 0, 0, 0.14),0px 4px 20px 3px rgba(0, 0, 0, 0.12)",
          "0px 7px 8px -4px rgba(0, 0, 0, 0.2),0px 12px 17px 2px rgba(0, 0, 0, 0.14),0px 5px 22px 4px rgba(0, 0, 0, 0.12)",
          "0px 7px 8px -4px rgba(0, 0, 0, 0.2),0px 13px 19px 2px rgba(0, 0, 0, 0.14),0px 5px 24px 4px rgba(0, 0, 0, 0.12)",
          "0px 7px 9px -4px rgba(0, 0, 0, 0.2),0px 14px 21px 2px rgba(0, 0, 0, 0.14),0px 5px 26px 4px rgba(0, 0, 0, 0.12)",
          "0px 8px 9px -5px rgba(0, 0, 0, 0.2),0px 15px 22px 2px rgba(0, 0, 0, 0.14),0px 6px 28px 5px rgba(0, 0, 0, 0.12)",
          "0px 8px 10px -5px rgba(0, 0, 0, 0.2),0px 16px 24px 2px rgba(0, 0, 0, 0.14),0px 6px 30px 5px rgba(0, 0, 0, 0.12)",
          "0px 8px 11px -5px rgba(0, 0, 0, 0.2),0px 17px 26px 2px rgba(0, 0, 0, 0.14),0px 6px 32px 5px rgba(0, 0, 0, 0.12)",
          "0px 9px 11px -5px rgba(0, 0, 0, 0.2),0px 18px 28px 2px rgba(0, 0, 0, 0.14),0px 7px 34px 6px rgba(0, 0, 0, 0.12)",
          "0px 9px 12px -6px rgba(0, 0, 0, 0.2),0px 19px 29px 2px rgba(0, 0, 0, 0.14),0px 7px 36px 6px rgba(0, 0, 0, 0.12)",
          "0px 10px 13px -6px rgba(0, 0, 0, 0.2),0px 20px 31px 3px rgba(0, 0, 0, 0.14),0px 8px 38px 7px rgba(0, 0, 0, 0.12)",
          "0px 10px 13px -6px rgba(0, 0, 0, 0.2),0px 21px 33px 3px rgba(0, 0, 0, 0.14),0px 8px 40px 7px rgba(0, 0, 0, 0.12)",
          "0px 10px 14px -6px rgba(0, 0, 0, 0.2),0px 22px 35px 3px rgba(0, 0, 0, 0.14),0px 8px 42px 7px rgba(0, 0, 0, 0.12)",
          "0px 11px 14px -7px rgba(0, 0, 0, 0.2),0px 23px 36px 3px rgba(0, 0, 0, 0.14),0px 9px 44px 8px rgba(0, 0, 0, 0.12)",
          "0px 11px 15px -7px rgba(0, 0, 0, 0.2),0px 24px 38px 3px rgba(0, 0, 0, 0.14),0px 9px 46px 8px rgba(0, 0, 0, 0.12)",
        ],
        "shape": Object {
          "borderRadius": 4,
        },
        "spacing": Object {
          "unit": 8,
        },
        "styles": Object {
          "buttonStyles": Object {
            "contained": Object {
              "&$contained:focus": Object {
                "backgroundColor": "#8849A5",
              },
              "&$contained:hover": Object {
                "backgroundColor": "#491760",
              },
              "backgroundColor": "#632B7D",
              "color": "white",
            },
            "outlined": Object {
              "&$outlined:focus": Object {
                "backgroundColor": "#F0F0F0",
              },
              "&$outlined:hover": Object {
                "backgroundColor": "#F7F7F7",
              },
              "backgroundColor": "white",
              "borderStyle": "hidden solid solid solid",
              "boxShadow": "0px 1px #C1C1C1",
              "color": "#D4436B",
            },
          },
          "classes": Object {
            "h1Style": "f2 roboto-light gray_01",
            "h2Style": "f3 roboto-light gray_01",
            "h3Style": "f3 roboto-light white",
            "h4Style": "f5 roboto-regular white",
            "h5Style": "f5 roboto-light white",
            "h6Style": "f2 brandon-regular white",
          },
          "colors": Object {
            "aqua": "#8ACAC7",
            "bgFooter": "#383838",
            "cobalt": "#0880C4",
            "fairy": "#B6DDDC",
            "gray_01": "#636363",
            "gray_02": "#9C9C9C",
            "gray_03": "#D4D4D4",
            "gray_04": "#F0F0F0",
            "green": "#94BA1D",
            "lightGreen": "#7FBC42",
            "pink": "#D2436C",
            "purple": "#632B7D",
            "red": "#BA2B31",
            "see": "#347A7C",
            "seeGradient": "linear-gradient(26deg, #347A7C, #8ACAC7)",
            "siren": "#019193",
            "turquise": "#5EB8C0",
            "white": "#ffffff",
            "yellow": "#F8DB02",
          },
          "fonts": Object {
            "body1": Object {
              "color": "#636363",
              "fontFamily": "Roboto Light, sans-serif",
              "fontSize": "16px",
            },
            "body2": Object {
              "color": "#636363",
              "fontFamily": "Roboto Regular, sans-serif",
              "fontSize": "16px",
            },
            "button": Object {
              "color": "#632B7D",
              "fontFamily": "Roboto Bold, sans-serif",
              "fontSize": "14px",
            },
            "caption": Object {
              "color": "#9C9C9C",
              "fontFamily": "Roboto Regular, sans-serif",
              "fontSize": "14px",
            },
            "display1": Object {
              "color": "#D2436C",
              "fontFamily": "Roboto Regular, sans-serif",
              "fontSize": "16px",
            },
            "display2": Object {
              "color": "#7FBC42",
              "fontFamily": "Roboto Regular, sans-serif",
              "fontSize": "16px",
            },
            "display3": Object {
              "color": "#F8DB02",
              "fontFamily": "Roboto Regular, sans-serif",
              "fontSize": "16px",
            },
            "display4": Object {
              "color": "#0880C4",
              "fontFamily": "Roboto Regular, sans-serif",
              "fontSize": "16px",
            },
            "h1": Object {
              "color": "#636363",
              "fontFamily": "Roboto Light, sans-serif",
              "fontSize": "36px",
            },
            "h2": Object {
              "color": "#636363",
              "fontFamily": "Roboto Light, sans-serif",
              "fontSize": "24px",
              "lineHeight": 1.5,
            },
            "h3": Object {
              "color": "#ffffff",
              "fontFamily": "Roboto Light, sans-serif",
              "fontSize": "24px",
            },
            "h4": Object {
              "color": "#ffffff",
              "fontFamily": "Roboto Regular, sans-serif",
              "fontSize": "16px",
            },
            "h5": Object {
              "color": "#ffffff",
              "fontFamily": "Roboto Light, sans-serif",
              "fontSize": "16px",
            },
            "h6": Object {
              "color": "#ffffff",
              "fontFamily": "Brandon Regular, sans-serif",
              "fontSize": "30px",
            },
            "headline": Object {
              "color": "#ffffff",
              "fontFamily": "Brandon Regular, sans-serif",
              "fontSize": "20px",
            },
            "inherit": Object {},
            "overline": Object {
              "color": "#9C9C9C",
              "fontFamily": "Roboto Bold, sans-serif",
              "fontSize": "16px",
            },
            "srOnly": Object {},
            "subheading": Object {
              "color": "#ffffff",
              "fontFamily": "Roboto Regular, sans-serif",
              "fontSize": "12px",
            },
            "subtitle1": Object {
              "color": "#ffffff",
              "fontFamily": "Roboto Bold, sans-serif",
              "fontSize": "16px",
            },
            "subtitle2": Object {
              "color": "#ffffff",
              "fontFamily": "Roboto Bold, sans-serif",
              "fontSize": "14px",
            },
            "title": Object {
              "color": "#636363",
              "fontFamily": "Roboto Light, sans-serif",
              "fontSize": "28px",
            },
          },
        },
        "transitions": Object {
          "create": [Function],
          "duration": Object {
            "complex": 375,
            "enteringScreen": 225,
            "leavingScreen": 195,
            "short": 250,
            "shorter": 200,
            "shortest": 150,
            "standard": 300,
          },
          "easing": Object {
            "easeIn": "cubic-bezier(0.4, 0, 1, 1)",
            "easeInOut": "cubic-bezier(0.4, 0, 0.2, 1)",
            "easeOut": "cubic-bezier(0.0, 0, 0.2, 1)",
            "sharp": "cubic-bezier(0.4, 0, 0.6, 1)",
          },
          "getAutoHeightDuration": [Function],
        },
        "typography": Object {
          "body1": Object {
            "color": "#636363",
            "fontFamily": "Roboto Light, sans-serif",
            "fontSize": "16px",
            "fontWeight": 400,
            "lineHeight": "1.46429em",
          },
          "body1Next": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "lineHeight": 1.5,
          },
          "body2": Object {
            "color": "#636363",
            "fontFamily": "Roboto Regular, sans-serif",
            "fontSize": "16px",
            "fontWeight": 500,
            "lineHeight": "1.71429em",
          },
          "body2Next": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "lineHeight": 1.5,
          },
          "button": Object {
            "color": "#632B7D",
            "fontFamily": "Roboto Bold, sans-serif",
            "fontSize": "14px",
            "fontWeight": 500,
            "textTransform": "uppercase",
          },
          "buttonNext": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 500,
            "lineHeight": 1.5,
            "textTransform": "uppercase",
          },
          "caption": Object {
            "color": "#9C9C9C",
            "fontFamily": "Roboto Regular, sans-serif",
            "fontSize": "14px",
            "fontWeight": 400,
            "lineHeight": "1.375em",
          },
          "captionNext": Object {
            "color": "rgba(0, 0, 0, 0.87)",
            "fontFamily": "'Roboto Light', 'sans-serif'",
            "fontSize": "NaNrem",
            "fontWeight": 400,
            "lineHeight": 1.66,
          },
          "color": "#636363",
          "display1": Object {
            "color": "#D2436C",
            "fontFamily": "Roboto Regular, sans-serif",
            "fontSize": "16px",
            "fontWeight": 400,
            "lineHeight": "1.20588em",
          },
          "display2": Object {
            "color": "#7FBC42",
            "fontFamily": "Roboto Regular, sans-serif",
            "fontSize": "16px",
            "fontWeight": 400,
            "lineHeight": "1.13333em",
            "marginLeft": "-.02em",
          },
          "display3": Object {
            "color": "#F8DB02",
            "fontFamily": "Roboto Regular, sans-serif",
            "fontSize": "16px",
            "fontWeight": 400,
            "letterSpacing": "-.02em",
            "lineHeight": "1.30357em",
            "marginLeft": "-.02em",
          },
          "display4": Object {
            "color": "#0880C4",
            "fontFamily": "Roboto Regular, sans-serif",
            "fontSize": "16px",
            "fontWeight": 300,
            "letterSpacing": "-.04em",
            "lineHeight": "1.14286em",
            "marginLeft": "-.04em",
          },
          "fontFamily": "'Roboto Light', 'sans-serif'",
          "fontSize": "1rem",
          "fontWeightLight": 300,
          "fontWeightMedium": 500,
          "fontWeightRegular": 400,
          "h1": Object {
            "color": "#636363",
            "fontFamily": "Roboto Light, sans-serif",
            "fontSize": "36px",
            "fontWeight": 300,
            "lineHeight": 1,
          },
          "h2": Object {
            "color": "#636363",
            "fontFamily": "Roboto Light, sans-serif",
            "fontSize": "24px",
            "fontWeight": 300,
            "lineHeight": 1.5,
          },
          "h3": Object {
            "color": "#ffffff",
            "fontFamily": "Roboto Light, sans-serif",
            "fontSize": "24px",
            "fontWeight": 400,
            "lineHeight": 1.04,
          },
          "h4": Object {
            "color": "#ffffff",
            "fontFamily": "Roboto Regular, sans-serif",
            "fontSize": "16px",
            "fontWeight": 400,
            "lineHeight": 1.17,
          },
          "h5": Object {
            "color": "#ffffff",
            "fontFamily": "Roboto Light, sans-serif",
            "fontSize": "16px",
            "fontWeight": 400,
            "lineHeight": 1.33,
          },
          "h6": Object {
            "color": "#ffffff",
            "fontFamily": "Brandon Regular, sans-serif",
            "fontSize": "30px",
            "fontWeight": 500,
            "lineHeight": 1.6,
          },
          "headline": Object {
            "color": "#ffffff",
            "fontFamily": "Brandon Regular, sans-serif",
            "fontSize": "20px",
            "fontWeight": 400,
            "lineHeight": "1.35417em",
          },
          "inherit": Object {},
          "overline": Object {
            "color": "#9C9C9C",
            "fontFamily": "Roboto Bold, sans-serif",
            "fontSize": "16px",
            "fontWeight": 400,
            "lineHeight": 2.66,
            "textTransform": "uppercase",
          },
          "pxToRem": [Function],
          "round": [Function],
          "srOnly": Object {},
          "subheading": Object {
            "color": "#ffffff",
            "fontFamily": "Roboto Regular, sans-serif",
            "fontSize": "12px",
            "fontWeight": 400,
            "lineHeight": "1.5em",
          },
          "subtitle1": Object {
            "color": "#ffffff",
            "fontFamily": "Roboto Bold, sans-serif",
            "fontSize": "16px",
            "fontWeight": 400,
            "lineHeight": 1.75,
          },
          "subtitle2": Object {
            "color": "#ffffff",
            "fontFamily": "Roboto Bold, sans-serif",
            "fontSize": "14px",
            "fontWeight": 500,
            "lineHeight": 1.57,
          },
          "title": Object {
            "color": "#636363",
            "fontFamily": "Roboto Light, sans-serif",
            "fontSize": "28px",
            "fontWeight": 500,
            "lineHeight": "1.16667em",
          },
          "useNextVariants": false,
        },
        "zIndex": Object {
          "appBar": 1100,
          "drawer": 1200,
          "mobileStepper": 1000,
          "modal": 1300,
          "snackbar": 1400,
          "tooltip": 1500,
        },
      }
    }
  >
    <WithStyles(RentabilityBar)
      firstTitle="Rentabilidade em R$"
      firstValue="100.000"
      secondTitle="Rentabilidade em %"
      secondValue="0"
      thirthTitle="cdi"
      thirthValue="100"
    />
  </MuiThemeProviderOld>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "firstTitle": "Rentabilidade em R$",
      "firstValue": "100.000",
      "secondTitle": "Rentabilidade em %",
      "secondValue": "0",
      "thirthTitle": "cdi",
      "thirthValue": "100",
    },
    "ref": null,
    "rendered": null,
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "firstTitle": "Rentabilidade em R$",
        "firstValue": "100.000",
        "secondTitle": "Rentabilidade em %",
        "secondValue": "0",
        "thirthTitle": "cdi",
        "thirthValue": "100",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
